tests/test/import/test.sh

```diff
@@ -56,6 +56,20 @@ rlJournalStart
     rlPhaseStartTest 'Multihost'
         rlRun "tmt tests ls . --filter 'tag:multihost' | tee output"
         rlAssertGrep "/parent/child" 'output'
+        rlRun 'tmt test show | tee output'
+        rlAssertGrep "tag multihost" 'output'
```

Comment:
I guess these two lines just duplicate the previous two lines, right? We just check that the `multihost` tag has been correctly assigned.